### DISCUSSION TITLE
Has anyone added TSparticles to Wordpress?

### DISCUSSION THREAD

**Discussion by RichardUUU** [matteobruni/tsparticles#2185] [General]
Any ideas welcome on how to add this to a Wordpress site.

Thanks, Richard

**Comment by matteobruni** [comment]
Hi @RichardUUU,

There's this addon that uses tsParticles for particles animations: https://premiumaddons.com/particles-section-addon-for-elementor-page-builder/

I gave them all the support they needed, so I think this is the better solution.

Otherwise adding the script manually could be another good solution (https://www.ostraining.com/blog/wordpress/custom-js/ this could be helpful I think)

**Comment by RichardUUU** [comment]
Thanks @matteobruni, 

I used Elementor previously, but have moved on to Beaver Builder.

**Reply by matteobruni** [reply]
@RichardUUU I found that this plugin uses tsParticles: https://www.ultimatebeaver.com/modules/particle-backgrounds/ (don't know the version, but you can ask them support)

**Reply by RichardUUU** [reply]
That's perfect. Thanks for checking on that.

**Comment by tamagit80** [comment]
Hi, you could also enqueue the script in the function.php

Function.php
```
function add_yourTheme_scripts() {
    wp_register_script( 'tsparticle', 'https://cdn.jsdelivr.net/npm/tsparticles@1.35.1/tsparticles.min.js');
    wp_register_script( 'particleconfig', get_stylesheet_directory_uri() . '/yourPath/particleConfig.js', array(), false, true );
    wp_enqueue_script( 'tsparticle');
    wp_enqueue_script( 'particleconfig');
}
add_action( 'wp_enqueue_scripts', 'add_yourTheme_scripts' );
```

and

Config file - particleConfig
```
tsParticles.load("your DIV id", {
  "background": {...
```

Ciao

**Reply by RichardUUU** [reply]
That's very helpful. I know how to add to function.php. Where would I put particleConfig?

**Reply by NinjaAtWork** [reply]
Hi , i'm stuck here , can you complete this way by a finished example?please
how can i assign the DIV id to page elements by css? 
thanks @matteobruni  @tamagit80

**Comment by tamagit80** [comment]
hi, wherever you want.
ex: Your theme folder/js/particleConfig.js so the path in function will be "/js/particleConfig.js".
Hope it helps ciao
